sway/tree/view.c

```diff
@@ -475,3 +493,127 @@ void view_child_destroy(struct sway_view_child *child) {
 		free(child);
 	}
 }
+
+static char *parse_title_format(struct sway_view *view) {
+	if (!view->title_format || strcmp(view->title_format, "%title") == 0) {
+		return strdup(view_get_title(view));
```

Comment:
`strdup` will segfault if `view_get_title` returns NULL

Reply:
`parse_title_format` is only called if the title is not NULL. Should I still put a check in here?